Task_Prototype/index.js

```diff
@@ -1,22 +1,166 @@
-//Write code for the 1st task below this
-
-
-//Write your code above this line
-//Write code for the 2nd task below this
-
-
-//Write your code above this line
-//Write code for the 3rd task below this
-
-
-//Write your code above this line
-//Write code for the 4th task below this
-
-
-//Write your code above this line
-//Write code for the 5th task below this
-
-
-//Write your code above this line
-
-
+//Write code for the 1st task below this
+
+function Person(name, age) {
+
+  this.name = name;
+  this.age = age;
+
+  
+  this.greet = function() {
+    console.log(`Hello, my name is ${this.name} and I am ${this.age} years old.`);
+  };
+}
+
+
+const person1 = new Person('Omparth', 21);
+
+
+person1.greet();
+//Write your code above this line
+//Write code for the 2nd task below this
+  function Person(name, age) {
+ 
+  this.name = name;
+  this.age = age;
+
+  this.greet = function() {
+    console.log(`Hello, my name is ${this.name} and I am ${this.age} years old.`);
+  };
+}
+
+
+function Student(name, age, studentId) {
+ 
+  Person.call(this, name, age);
+
+
+  this.studentId = studentId;
+
+
+  this.study = function() {
+    console.log(`${this.name} with student ID ${this.studentId} is studying.`);
+  };
+}
+
+
+Student.prototype = Object.create(Person.prototype);
+
+
+const student1 = new Student('parth', 21, 'S123');
+
+
+student1.greet();
+
+
+student1.study();
+
+//Write your code above this line
+//Write code for the 3rd task below this
+function Person(name, age) {
+  this.name = name;
+  this.age = age;
+}
+
+
+Person.prototype.introduce = function() {
+  console.log(`Hello, my name is ${this.name} and I am ${this.age} years old.`);
+};
+
+
+function Student(name, age, studentId) {
+  Person.call(this, name, age);
+  this.studentId = studentId;
+}
+
+
+Student.prototype = Object.create(Person.prototype);
+

```

Comment:
The 'greet' method in line 97 of the Person class and line 111 of the Student class should use consistent wording in their console.log statements.